lib/matplotlib/backends/backend_ps.py

```diff
@@ -374,6 +369,37 @@ def get_image_magnification(self):
         """
         return self.image_magnification
 
+    def _convert_path(self, path, transform, clip=False, simplify=None):
+        if clip:
+            clip = (0.0, 0.0, self.width * 72.0, self.height * 72.0)
+        else:
+            clip = None
+        return _path.convert_to_string(
+            path, transform, clip, simplify, None,
+            6, [b"m", b"l", b"", b"c", b"cl"], True).decode("ascii")
+
+    def _get_clip_cmd(self, gc):
+        clip = []
+        rect = gc.get_clip_rectangle()
+        if rect:
```

Comment:
Is this change from `None`-check to `bool`-check intentional? Was `if bbox is not None` before (l.408).

Reply:
Previously one of the checks was `if cliprect` (in `_draw_ps`) and the other was `if bbox is None` (in `_get_clip_path`).  They are equivalent as the cliprect is `Optional[Bbox]`.

Reply:
I'd prefer checking explicitly for `None`.

I was starting to wonder if we have special bboxes that are falsy, similar to an empty list.

Reply:
sure